packages/toast/locales/nb/messages.po

```diff
@@ -0,0 +1,26 @@
+msgid ""
+msgstr ""
+"POT-Creation-Date: 2024-01-08 14:36+0100\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: @lingui/cli\n"
+"Language: nb\n"
+
+#. Default screenreader message for error in toast component
+#. js-lingui-explicit-id
+#: packages/toast/toast.js:112
+msgid "toast.aria.error"
+msgstr ""
+
+#. Default screenreader message for successful in toast component
+#. js-lingui-explicit-id
+#: packages/toast/toast.js:118
+msgid "toast.aria.successful"
+msgstr ""
+
+#. Default screenreader message for warning in toast component
+#. js-lingui-explicit-id
+#: packages/toast/toast.js:106
+msgid "toast.aria.warning"
+msgstr ""
```

Comment:
Unless you want to wait with publishing a new version until all translations are in place, you could manually add the translations we had before and ship it without breaking accessibility on finn.no

Reply:
Added Norwegian translation in this commit: [3ac496e](https://github.com/warp-ds/elements/pull/123/commits/3ac496e7c23d825159e03800f13154265689f3ee)

Reply:
But do I also need to manually change the messages.mjo file as well for Norwegian?

Reply:
Once you update .po files, you run `messages:compile` again to see mjs files being updated.

Reply:
Oh right! 😄 Ok! 👍